infra/base-images/base-runner/Dockerfile

```diff
@@ -17,8 +17,10 @@
 FROM gcr.io/oss-fuzz-base/base-image
 MAINTAINER mike.aizatsky@gmail.com
 RUN apt-get install -y zip file libunwind8 binutils libblocksruntime0 \
-    fonts-dejavu python3 libcap2 wget
-COPY bad_build_check coverage coverage_helper.py download_corpus llvm-cov \
+    fonts-dejavu python3 python3-pip libcap2 wget git
```

Comment:
alpha order

Reply:
Done. Can do the same for other files (probably a separate CL would be better).